axelrod/ecosystem.py

```diff
@@ -0,0 +1,38 @@
+class Ecosystem(object):
+    """Create an ecosystem based on the payoff matrix from an Axelrod tournament."""
+
+    def __init__(self, pmatrix):
+
+        self.pmatrix = pmatrix
```

Comment:
> Ah: you're not re-running the tournament each time... You're running it once and using the outcome... (Currently throwing that around in my head... Not what I would have done off the bat...)

Exactly. There is no point in running it every time, since the average payoffs will be the same approximately. An efficient option would be to also calculate payoff variances and generate a payoff from a normal distribution at each turn.